### DISCUSSION TITLE
My public repository does not appear in "Repository" search results anymore

### DISCUSSION THREAD

**Discussion by olshevski** [community/community#87236] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have a small open-source library https://github.com/olshevski/compose-navigation-reimagined. Out of pure interest I recently tried searching it with GitHub search and was really surprised that it doesn't appear in search results anymore. I tried searching by general keywords (like `compose navigation` or `compose navigation android`) and by full library name (`compose navigation reimagined` and `compose-navigation-reimagined`), but it never appeared in `Repositories` results.

It definitely worked at some point previously, but not anymore. I also have a suspicion that this is the reason why the library dropped in attention from developers, as it is simply not discoverable through search anymore.

Is there any reason why it might have been excluded from search results? Could it be a bug in GitHub search?

**Comment by anuragjain-git** [comment]
it hasn't index by github yet ig, were you able to search your repo before ? It also dosen't shows up in https://github.com/olshevski?tab=repositories

**Reply by olshevski** [reply]
Yes, it was appearing in the search results last year. I was checking it occasionally to see if it appeared on the first page of search results. Just enjoyed it slowly growing

**Comment by faridnec** [comment]
Possibly a bug, also there could be several reasons why your repository is not showing up in GitHub search results. Sometimes, GitHub search results may take some time to update, and there could be a delay in indexing new or updated repositories. Wait for some time to see if there's a delay in indexing. View this [link](https://github.com/orgs/community/discussions/18624)

**Comment by olshevski** [comment]
It finally reappeared in the search results. No idea what actually triggered the indexing. I tried searching the source code specifying the `repo:` qualifier with my repository name. Someone suggested that it might trigger the indexing, so it might be the case.

**Reply by anuragjain-git** [reply]
yes someone in #18624 suggested searching reponame can trigger indexing. Did you find you what made it disappear from the search results?
